### PR TITLE
Fix: PHP Unit test issues 

### PR DESCRIPTION
**Pull request by pritesh-lr**
The purpose of this PR is to fix PHP Unit test cases issues and added some new cases.

### CODE REVIEW

File: Test/Unit/Gateway/Request/AuthorizeRequestTest.php [hunk 1]
```diff
@@ -48,14 +61,22 @@ protected function setUp(): void
...
             ->getMock();
-        $this->authorizationRequest = new AuthorizationRequest($this->payment);
+        $this->helper = $this->getMockBuilder(Data::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->config = $this->getMockBuilder(Config::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->authorizationRequest = new AuthorizationRequest($this->helper, $this->config);
+        // $this->authorizationRequest = new AuthorizationRequest($this->payment);
```
Comment (mstein-ck) [inline Test/Unit/Gateway/Request/AuthorizeRequestTest.php:71]: Can this comment be removed? Is there a reason to keep it?

File: Test/Unit/Gateway/Request/CaptureRequestTest.php [hunk 2]
```diff
@@ -11,51 +11,104 @@
...
         $amount = "1.00";
         $storeId = 1;
         $refnum = '23443535';
+        $currencyCode = 'USD';
+        $invoiceId = 1001;
 
         $expectation = [
             'xCommand' => 'cc:capture',
-            'xAmount' => $amount,
+            'xAmount'   => $this->helper->formatPrice($amount),
```
Comment (mstein-ck) [inline Test/Unit/Gateway/Request/CaptureRequestTest.php:65]:
```suggestion
            'xAmount' => $this->helper->formatPrice($amount),
```

File: Test/Unit/Gateway/Request/RefundRequestTest.php [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+        $this->orderMock->expects(static::any())
+            ->method('getGrandTotalAmount')
+            ->willReturn($amount);
+
+        static::assertEquals(
+            $expectation,
+            $this->refundRequest->build($buildSubject)
+        );
+        // $this->refundRequest->build($buildSubject);
```
Comment (mstein-ck) [inline Test/Unit/Gateway/Request/RefundRequestTest.php:108]: Is this comment needed?

Comment (pritesh-lr) [reply Test/Unit/Gateway/Request/RefundRequestTest.php]: No, it's not needed. I have removed that comment code and pushed it.

File: Test/Unit/Gateway/Request/AuthorizeRequestTest.php (hunk 1 above)
Comment (pritesh-lr) [reply Test/Unit/Gateway/Request/AuthorizeRequestTest.php]: Yes, This comment code is removed now, and commit the changes.

**Review by mstein-ck — APPROVED** [review]

**Pull request merged**